Vic2ToHoI4Tests.vcxproj

```diff
@@ -869,22 +869,14 @@
       <DestinationFolders Condition="'$(Configuration)|$(Platform)'=='Release|x64'">$(OutDir)/Configurables</DestinationFolders>
     </CopyFileToFolders>
   </ItemGroup>
-  <ItemGroup>
-    <CopyFileToFolders Include="data\test_files\Buildings\01_landmark_buildings.txt">
-      <DestinationFolders Condition="'$(Configuration)|$(Platform)'=='Release|x64'">$(OutDir)/HoI4Windonws/common/buildings</DestinationFolders>
-    </CopyFileToFolders>
-    <CopyFileToFolders Include="data\test_files\Buildings\LandmarksMappings.txt">
-      <DestinationFolders Condition="'$(Configuration)|$(Platform)'=='Release|x64'">$(OutDir)/Configurables</DestinationFolders>
-    </CopyFileToFolders>
-  </ItemGroup>
```

Comment:
Did these files get removed accidentally?

Reply:
There was another instance of them higher in the file. I had assumed I added the second instance as a bad merge, but it was pre-existing.